style.css

```diff
@@ -303,3 +327,159 @@ input {
   color: #42526e;
   margin-left: 25px;
 }
+
+/* ---------------Mobile Vesrion Ends Here--------------- */
+
+/* -----------------------Desktop Vesrion Started----------------------- */
+
+@media screen and (min-width: 768px) {
+  #headline-img-desktop {
+    display: block;
+    width: 100%;
+  }
+
+  .navbar {
+    display: block;
+    margin-left: 41%;
+  }
```

Comment:
- [ ] [OPTIONAL] Your site overflows. Kindly fix this 🙏

Reply:
Thank for suggestion it is also fixed now.